easybuild/easyconfigs/f/FastQ_Screen/FastQ_Screen-use-EB-Perl.patch

```diff
@@ -0,0 +1,10 @@
+diff -u FastQ-Screen-0.14.1/fastq_screen FastQ-Screen-0.14.1.new/fastq_screen
```

Comment:
Can we use `fix_perl_shebang_for` instead of the patch?

If not, then please add a description / author to the patch.